config/rbac/hive_frontend_role.yaml

```diff
@@ -47,6 +47,7 @@ rules:
   - syncidentityproviders
   - selectorsyncsets
   - syncsets
+  - clusterdeprovisionrequests
```

Comment:
@jhernand @vkareh new requests API being added to frontend role here.

Reply:
Great, thanks @dgoodwin.